pkg/apps/copier.go

```diff
@@ -114,7 +140,8 @@ func (f *aferoCopier) Copy(stat os.FileInfo, src io.Reader) (err error) {
 	if !f.started {
 		panic("copier should call Start() before Copy()")
 	}
-	fullpath := path.Join(f.appDir, stat.Name())
+
+	fullpath := path.Join(f.appDir, stat.Name()) + ".gz"
```

Comment:
Does travis build fail because we are now storing a file `app.tar.gz` for konnectors but still looking for `app.tar` (just a guess)?

Reply:
Or is it just the konnectors _tests_ are still looking for `app.tar`?

Reply:
The problem was in the tests that I had to adapt.